src/main/java/com/board/tddboard/dao/BoardDao.java

```diff
@@ -0,0 +1,12 @@
+package com.board.tddboard.dao;
+
+import com.board.tddboard.domain.Board;
+
+public interface BoardDao {
+
+    public Board save(Board board);
+    public Board findById(Long id);
+    public void removeById(Long id);
+    public void updateById(Long id);
```

Comment:
```suggestion
    public void removeById(Long id);
    public void updateById(Long id);
```

삭제나 수정이후 `void` 로 리턴하는 것보다 `int` 나 `boolean` 으로 리턴해서 유효성 확인까지 하는건 어떨까요